app/commands/user/reputation_token/mark_as_seen.rb

```diff
@@ -0,0 +1,12 @@
+class User::ReputationToken::MarkAsSeen
+  include Mandate
+
+  initialize_with :token
+
+  def call
+    return if token.seen?
```

Comment:
This is a micro optimization to prevent unnecessary cache resetting. Is this okay or should I use the return value of `seen!` (or some other state)

Reply:
This is good.